src/transformStateWithClones.js

```diff
@@ -65,7 +65,43 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const resultObj = [];
```

Comment:
Не скорочуй назви змінних
```suggestion
  const states = [];
```